### ISSUE TITLE
NPM package does not contain distribution

### ISSUE THREAD

**Issue by jochenberger** [luqin/react-jstree#1]
```
$ npm install react-jstree
react-jstree@1.0.0 node_modules/react-jstree
├── react-dom@0.14.2
└── react@0.14.2 (envify@3.4.0, fbjs@0.3.2)
$ ls node_modules/react-jstree/
config.js  gulpfile.js  node_modules  package.json  README.md  src
$ ls node_modules/react-jstree/src/
index.js
$ cat node_modules/react-jstree/src/index.js 
$
```

The `dist` directory is missing from the package and `src/index.js` is empty.

**Comment by luqin** [comment]
@jochenberger 
Yes. Because I have not implement any feafure in this repo. I'm think about how to rewrite jstree components or wrap jstree with react. Do you have any idea?

**Comment by jochenberger** [comment]
Wrapping might be hard when it comes to updating an existing tree. jsTree does not preserve the `data-react-id` on `ul` elements (https://github.com/vakata/jstree/issues/1271) and React chokes on that pretty hard. And I'm not aware of a proper API call that updates an existing tree with new data.
You could diff the JSON manually and convert it into proper API calls on the jsTree instance, but it might be easier to do a full rewrite.

**Comment by conor909** [comment]
This repo would be great. Is there plans for it to be developed  and maintained further?

**Comment by Data-Meister** [comment]
OH! Is this not working at all yet?

**Comment by luqin** [comment]
@Data-Meister Congratulations,you got it

**Comment by stonecharioteer** [comment]
@luqin Hey, did you make any progress with this?
